<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="buttonBox">
      <property name="styleSheet">
       <string notr="true">background-color:#EDF5F3;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="taskButton">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>任务</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">functionbuttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="listButton">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>列表</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">functionbuttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calendarButton">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>日历</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">functionbuttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="noteButton">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>笔记</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">functionbuttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="classificationButton">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>分类</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">functionbuttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settingButton">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>设置</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">functionbuttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QTableView" name="tasklist">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>461</width>
          <height>501</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>-1</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableView::item {
    height: 60px;
}
QTableView {
	font: 25 9pt &quot;Microsoft YaHei&quot;;
	border: 15px solid white; /* 设置边框的大小，样式，颜色 */
	border-radius: 10px;
}
QTableView::item:hover {
	background-color: transparent;
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}
QTableView::item:selected {
	background-color: transparent;
	color: black;
	padding: 10px;
	border-left: 3px solid black;
}
QTableView { font-size: 14px; }</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>130</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>40</number>
        </attribute>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>20</y>
          <width>191</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="name_1">
           <property name="text">
            <string>项目名称</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeinday_1">
           <property name="text">
            <string>今日时长</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="remaintime_1">
           <property name="text">
            <string>剩余时长</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="totaltime_1">
           <property name="text">
            <string>总时长</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>300</y>
          <width>122</width>
          <height>158</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>新建</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QListWidget" name="todolist">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>117</width>
          <height>501</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="flow">
         <enum>QListView::TopToBottom</enum>
        </property>
        <item>
         <property name="text">
          <string>泡面</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="lastMonthButton">
            <property name="text">
             <string>上一个月</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="lastWeekButton">
            <property name="text">
             <string>上一周</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="datenow">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextWeekButton">
            <property name="text">
             <string>下一周</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextMonthButton">
            <property name="text">
             <string>下一个月</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="taskCalendar">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>85</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>35</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>250</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>笔记</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>330</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>分类</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>300</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="functionbuttonGroup"/>
 </buttongroups>
</ui>
